File: journaling-app-backend/clear_database.js
const sqlite3 = require('sqlite3').verbose();

// Initialize the database
const db = new sqlite3.Database('database.db');

db.serialize(() => {
    // Clear users table
    db.run('DELETE FROM users', (err) => {
        if (err) {
            console.error('Error clearing users table:', err.message);
        } else {
            console.log('Users table cleared.');
        }
    });

    // Clear journal_entries table
    db.run('DELETE FROM journal_entries', (err) => {
        if (err) {
            console.error('Error clearing journal_entries table:', err.message);
        } else {
            console.log('Journal entries table cleared.');
        }
    });
});

db.close();
console.log('Database cleared.');

File: backend/clear_database.py
import sqlite3

def clear_db():
    conn = sqlite3.connect('database.db')
    c = conn.cursor()
    c.execute('DELETE FROM journal_entries')
    c.execute('DELETE FROM users')
    conn.commit()
    conn.close()

if __name__ == "__main__":
    clear_db()
    print("All records cleared from the database.")
